# Define the goal
geometry_msgs/Pose pose
bool is_close
---
# Define the result
uint32 done
---
# Define a feedback message
float32 percent_complete
